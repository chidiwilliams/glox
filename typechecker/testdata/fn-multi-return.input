fun getName(): number {
	if (true) return 0;
	return "name";
}
