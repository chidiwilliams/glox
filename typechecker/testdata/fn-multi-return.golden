expected enclosing function to return 'number', but got 'string'
